app/presenters/document/record_presenter.rb

```diff
@@ -36,39 +36,30 @@ def web_resources
       end
     end
 
-    def salient_media_web_resources
+    def edm_object_thumbnails_edm_is_shown_by?
+      edm_is_shown_by.present? && edm_object.present? && (edm_object != edm_is_shown_by)
+    end
+
+    def salient_media_web_resource_presenters
```

Comment:
Assignment Branch Condition size for salient_media_web_resource_presenters is too high. [15.65/15]<br>Method has too many lines. [14/10]